app/cpfvalidator/validator.go

```diff
@@ -0,0 +1,140 @@
+// Contains functions for validating CPF, the Brazilian unique identifacation number for every person in the country.
+// CPFs must contain 11 numeric digits, being the last 2 for validation.
+// For the purpose of this validation, the first 9 digits can be considered random
+// (although there are rules related to the State in which it was emitted, for example).
+// The official refference can be found at http://sa.previdencia.gov.br/site/2015/07/rgrv_RegrasValidacao.pdf.
+package cpfvalidator
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+type Cpf string
+
+// These values pass the algorithm but are considered invalid.
+var knownInvalids = map[Cpf]struct{}{
+	"00000000000": {},
+	"11111111111": {},
+	"22222222222": {},
+	"33333333333": {},
+	"44444444444": {},
+	"55555555555": {},
+	"66666666666": {},
+	"77777777777": {},
+	"88888888888": {},
+	"99999999999": {},
+}
+
+// Runs CPF algorithm to check if CPF is valid. Accepts XXX.XXX.XXX-XX and XXXXXXXXXXX formats.
```

Comment:
É também um padrão (boa prática) começar o comentário que identifica uma func, var etc. com o nome dela. Nesse caso ficaria algo assim:
```suggestion
// IsValid runs CPF algorithm to check if CPF is valid. Accepts XXX.XXX.XXX-XX and XXXXXXXXXXX formats.
```

https://go.dev/blog/godoc